markdown/ReshadeGuides/setupreshade.md

```diff
@@ -1,116 +1,116 @@
-
+ 
 ### Picking the right depth buffer
 It might be the game uses multiple depth buffers that might be useful, or it switches depth buffers on the fly for the final image and therefore it looks like the
 depth buffer isn't available. For DirectX11 Reshade contains an additional tab the **DX11** tab, which shows all currently active depth buffers. To see which one
 you should select, first enable the *DisplayDepth* shader. You likely either see a white or a black screen. 
-
+ 
 When you go to the DX11 tab in the Reshade menu you can then pick the right depth buffer by checking its checkbox (or by trying the 
 *Copy depth before clearing* checkbox). If the screen becomes what it should be you picked the right depth buffer and you can use it with the techniques that depend on it. 
-
+ 
 ## Shader duplication
-
+ 
 There are cases where you would want to use one shader more than once (CanvasFog and StageDepthPlus shaders are popular in that regard). With most shaders all you need to do is replace the technique name in the shader code to a different one, but some shaders will need to change the name of a `namespace` or a texture name, so be aware of that. Be sure to check the shaders code for comments regarding duplicating the shader.
-
-## Limitations and gotchas
-There are some limitations with the current Reshade version (v4.2) which might cause some confusion. They're listed below.
-
-* When the game is an online game or actively performs network access, the depth buffer is disabled. Access to the depth buffer is 
-often required as a lot of effects use the depth buffer of the game. Reshade actively disables access to the depth buffer if it
-detects the game to be an online game (so there's network traffic on a regular basis). 
-If the depth buffer isn't available in the game, because of this, the shaders that depend on that will likely fail. E.g. using a 
-depth of field shader in a game like The Division is therefore not going to work: the depth buffer is disabled. 
-
+ 
+## Depth buffer on online games
+ 
+When the game is an online game or actively performs network access, the depth buffer is disabled. Access to the depth buffer is often required as a lot of effects use the depth buffer of the game. Reshade actively disables access to the depth buffer if it
+detects the game to be an online game (so there's network traffic on a regular basis). If the depth buffer isn't available in the game, because of this, the shaders that depend on that will likely fail.
+ 
+So if you are trying to use reshade in a game which uses the internet connection constantly (because of the DRM or because it's always online like The Division or Steep) you will have to use custom reshade .dlls builds to get depth buffer access, otherwise reshade will block it. You can find some of these .dlls [here](http://project7.rf.gd). Alternatively you can still use reshade without depth buffer access, but some shaders that use it like CinematicDof won't work.
```

Comment:
I didn't add a link to the debbie page as it's a page with executables and no real owner in sight. I know this is a grey area, but I think tools made by people we know is something different than a random on the internet. So I won't merge this.

Reply:
My apologies,I should have give it a proper thought.
Thanks for correction frans!